bin/pull_request_blaster_outer.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+
+require 'bundler/setup'
+require 'manageiq/release'
+require 'trollop'
+
+opts = Trollop.options do
+  opt :repo,    "The repo to update. If not passed, will try all repos in config/repos.yml. For example: --repo manageiq or --repo miq-test/sandbox", :type => :string
+  opt :branch,  "The target branch we want to open a pull request against.",                             :type => :string, :required => true
+  opt :script,  "The path to the script that will update the desired files. See the scripts directory.", :type => :string, :required => true
```

Comment:
It was much easier to let the user create a script in bash, ruby, etc. and just run it from here.  Inline sed or awk with escaping feels painful.  Plus, we can commit our common blasted PRs.